.npmignore

```diff
@@ -1,25 +1,9 @@
-.project
 .settings
-
-demos
-examples
-Gruntfile.js
-products
-
-src/components
-src/lib/fonts
-src/lib/jquery/plugins
-src/lib/jquery/ui
-
-src/framework/core/css
-src/framework/preferences
-
-tests/3rd-party-tests
-tests/component-tests
-tests/framework-tests
-tests/manual-tests
-tests/all-tests.html
-tests/lib/jquery-ui
-tests/lib/mockjax
-tests/lib/README
-tests/test-core/utils/js/DebugFocus.js
+.gitignore
+.jshintignore
+.jshintrc
```

Comment:
Should we leave the jshint files? They aren't really necessary for a build, but at least the .jshintrc file is used by a task in Gruntfile.js.